packages/cashier/src/pages/account-transfer/account-transfer-receipt/account-transfer-receipt.tsx

```diff
@@ -24,10 +24,10 @@ const AccountTransferReceipt = observer(({ onClose, history }: TAccountTransferR
     const { account_transfer } = useCashierStore();
     const { disableApp, enableApp } = ui;
     const { is_from_derivgo } = common;
-    const { is_pre_appstore, loginid, switchAccount } = client;
+    const { loginid, switchAccount } = client;
     const { receipt, resetAccountTransfer, selected_from, selected_to, setShouldSwitchAccount } = account_transfer;
 
-    const is_from_pre_appstore = is_pre_appstore && !location.pathname.startsWith(routes.cashier);
+    const is_from_pre_appstore = !location.pathname.startsWith(routes.cashier);
```

Comment:
this should be named `is_not_in_cashier` or something more appropriate

Reply:
aightss